gini/gini-vision-lib-ios
# Add errorview to multipage screen

Add error view in each page of the multipage screen.

### How to test
Open the app, select an invalid image and see the error in the multipage screen.

### Merging
Automatic
